aten/src/TH/THGenerateQUInt8Type.h

```diff
@@ -0,0 +1,22 @@
+#ifndef TH_GENERIC_FILE
```

Comment:
hm... what is the function from TH you still need? I propose ban on bringing new stuff into TH. I guess the right approach is to move whatever yo need outside of TH into native functions. @li-roy and @gchanan might have more context

Reply:
I need THStorage for qint types, I checked with Greg and Roy, Greg says we'll remove THStorage and Storage in the future so this is just temporary solution.

Reply:
But you need it only for set_, right? Maybe we can just migrate that method from TH (cc @li-roy)

Reply:
I think we decided to add quantized storage types for different reason, and it's still required. I don't recall the specific reason though, can you refresh my memory @jerryzh168

Reply:
One reason is we want to distinguish the quantized and non-quantized storage in frontend as well. Not sure whether that answers the question.